test/test_torch.py

```diff
@@ -12517,14 +12574,27 @@ def test_sinh_unary_mem_overlap(self):
     def test_cosh_unary_mem_overlap(self):
         self.unary_check_mem_overlap(lambda t: t.cosh_())
 
-    @unittest.expectedFailure
-    def test_lerp_mem_overlap(self):
-        start = torch.randn(1, device=device).expand(3, 3)
-        end = torch.randn(3, 3, device=device)
-        weight = torch.randn(3, 3, device=device)
+    @staticmethod
+    def _test_lerp_mem_self_overlap(self, device='cpu'):
+        start = torch.randn((1), device=device).expand(3, 3)
```

Comment:
(1) doesn't do anything, it is still 1. `(1,)` is the tuple with one element containing 1.

Reply:
addressed